docs/context/ref/InstrumentList.md

```diff
@@ -54,7 +54,7 @@ const instruments = {
     ]
 }
 
-fdc3.joinChannel('global')
+fdc3.joinUserChannel('global')
```

Comment:
This should be changed from `global` to another channel name, but can be handled separately from this PR.